README.md

```diff
@@ -157,6 +158,11 @@ If your font is not included in either of these locations you must include an hr
 
 Font support is a work in progress.
 
+## SmartObject
+
+The Photoshop APIs support creation of a SmartObject layer with Embedded Smart Object and Edit of the content of that Smart Object .
+When you edit and save changes to the smart object content, the edits appear in all linked instances of that Smart Object in the Photoshop document. Please see the api docs for more information .
+
```

Comment:
@khound my suggested edits:

- The Photoshop APIs currently support creating and editing of Embedded Smart Objects. Support for Linked Smart Objects support is forthcoming.

- In order to update an embedded smart object that is referenced by multiple layers you only need to update one of those layers, the effect will be reflected in all layers referencing the same smart object.. 

- Smart Object support is a work in progress. 

- Please see the api docs for more information.

Reply:
👍